manager/state/raft/raft.go

```diff
@@ -193,6 +189,7 @@ func NewNode(ctx context.Context, opts NewNodeOptions) *Node {
 		stopCh:              make(chan struct{}),
 		doneCh:              make(chan struct{}),
 		removeRaftCh:        make(chan struct{}),
+		stopped:             make(chan struct{}),
```

Comment:
isn't that was `doneCh` is for?

I thought in order to stop one would close `stopCh` then wait one `doneCh`